src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -4724,6 +4724,8 @@ int RGWRados::copy_obj(RGWObjectCtx& obj_ctx,
     src_attrs.erase(RGW_ATTR_OLH_VER);
   }
 
+  src_attrs.erase(RGW_ATTR_OBJ_REPLICATION_TRACE);
```

Comment:
should we clear RGW_ATTR_OBJ_REPLICATION_STATUS and RGW_ATTR_OBJ_REPLICATION_TIMESTAMP too?

Reply:
yes, done.

Reply:
RGW_ATTR_OBJ_REPLICATION_STATUS can go too